tests/providers/rushfiles/test_provider.py

```diff
@@ -245,6 +245,109 @@ async def test_validate_v1_path_root(self, provider):
 
     #     result = await provider.revalidate_path(path, file_name, True)
     #     assert result.name in path.name
+    
+class TestDelete:
+
+    @pytest.mark.asyncio
+    @pytest.mark.aiohttpretty
+    async def test_delete_file(self, provider, root_provider_fixtures):
+        item = root_provider_fixtures['file_metadata_delete_ok']
+        path = RushFilesPath('/Tasks.xlsx', _ids=(provider.share['id'], item['InternalName']))
+        url = provider._build_filecache_url(str(provider.share['id']), 'files', item['InternalName'])
+        url_body = json.dumps({
+                        "Data":{
+                            "Deleted": True
```

Comment:
Filecache does not return `Data.Deleted`. Are you thinking of `Data.ClientJournalEvent.RfVirtualFile.Deleted`?

https://filecache01.rushfiles.com/swagger/ui/index#!/File/File_DeleteFile

```json
{
  "Code": 0,
  "FileLock": {
    "IsLocked": true,
    "LockTime": 0,
    "LockedBy": "string"
  },
  "Data": {
    "ClientJournalEvent": {
      "TransmitId": "string",
      "UserOrigin": "string",
      "ClientJournalEventType": 0,
      "VShare": "string",
      "ShareTick": 0,
      "TransmitCounter": 0,
      "TransmitTimeout": "2021-11-24T20:47:18.130Z",
      "RfVirtualFile": {
        "ShareId": "string",
        "SubShareIds": [
          "string"
        ],
        "UploadUrl": "string",
        "UploadName": "string",
        "InternalName": "string",
        "ArchivedFullName": "string",
        "Tick": 0,
        "ShareTick": 0,
        "UpdateTick": 0,
        "CreateTick": 0,
        "ParrentId": "string",
        "NoOfChildren": 0,
        "EndOfFile": 0,
        "AllocationSize": 0,
        "CreationTime": "2021-11-24T20:47:18.130Z",
        "LastAccessTime": "2021-11-24T20:47:18.130Z",
        "LastWriteTime": "2021-11-24T20:47:18.130Z",
        "Attributes": 1,
        "Local": true,
        "User": "string",
        "Deleted": true,
        "IsOffline": true,
        "HasHistory": true,
        "FilehHash": "string",
        "PublicName": "string",
        "DeleteTimestamp": 0,
        "IsFile": true,
        "RetensionTimestamp": 0,
        "InRetension": true,
        "FileLock": {
          "IsLocked": true,
          "LockTime": 0,
          "LockedBy": "string",
          "DeviceId": "string"
        }
      },
      "CreateTick": 0,
      "DeviceId": "string"
    },
    "PotentialConflict": true
  },
  "Message": "string",
  "ResponseInfo": {
    "ResponseCode": 1000,
    "Reference": "string"
  }
}
```